tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/errors/BigQuerySqlError.java

```diff
@@ -5,17 +5,22 @@
 
 import lombok.Getter;
 
-/** The base class to represent the SQL Error generated by the BigQuery server. It should be resolved from
- * {@link BigQueryException}. */
+/**
+ * The base class to represent the SQL Error generated by the BigQuery server. It should be resolved
+ * from {@link BigQueryException}.
+ */
 @Getter
 public abstract class BigQuerySqlError {
 
-  BigQueryException errorSource;
-  /** The occurrence of the error at a query. It may be null if the error does not belong to any specific locations. */
-  Position errPos;
+  private final BigQueryException errorSource;
+  /**
+   * The occurrence of the error at a query. It may be null if the error does not belong to any
```

Comment:
Sorry for the nit comments... but why there would be new lines in between :D? similar with the above.

Reply:
After trying a few times of the auto format, I figure out I need to apply the auto format on each page. Thus, the old version is not formatted. 

I check the style guide, I think having a new line is encouraged.
https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line